src/toolbar.js

```diff
@@ -305,6 +314,7 @@ export class Toolbar extends LitElement {
         <div class="width-title">Line Width</div>
         <mwc-slider pin markers step="1" value="8" min="1" max="20" id="line-width-slider" @change="${this._lineWidthChanged}"></mwc-slider>
       </div>
+      <mwc-button slot="action" raised id="usi-clear-button">Clear</mwc-button>
```

Comment:
let's just call it clear-button. The demo works everywhere and that piece is not usi-specific :).

Reply:
updated patch